upload.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 2019-03-18
+ * Time: 20:35
+ */
+
+$target_dir = "uploads/";
+$target_file = $target_dir . basename($_FILES["fileToUpload"]["name"]);
+$uploadOk = 1;
+$imageFileType = strtolower(pathinfo($target_file,PATHINFO_EXTENSION));
+// Check if image file is a actual image or fake image
+if(isset($_POST["submit"])) {
+    $check = getimagesize($_FILES["fileToUpload"]["tmp_name"]);
+    if($check !== false) {
+        echo "File is an image - " . $check["mime"] . ".";
+        $uploadOk = 1;
+    } else {
+        echo "File is not an image.";
+        $uploadOk = 0;
+    }
+}
+// Check if file already exists
+if (file_exists($target_file)) {
+    echo "Sorry, file already exists.";
+    $uploadOk = 0;
+}
+// Check file size
+//if ($_FILES["fileToUpload"]["size"] > 500000) {
+//    echo "Sorry, your file is too large.";
+//    $uploadOk = 0;
+
+// Allow certain file formats
+//if($imageFileType != "jpg" && $imageFileType != "png" && $imageFileType != "jpeg"
+//    && $imageFileType != "gif" ) {
+//    echo "Sorry, only JPG, JPEG, PNG & GIF files are allowed.";
+//    $uploadOk = 0;
+//}
+// Check if $uploadOk is set to 0 by an error
+if ($uploadOk == 0) {
+    echo "Sorry, your file was not uploaded.";
```

Comment:
Sorry file exists and Sorry file was not uploaded appear same time ( to fix in a next task?)